new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMonitoringExporter.java

```diff
@@ -226,112 +256,380 @@ public static void initAll(@Namespaces(6) List<String> namespaces) {
     installMonitoringExporter();
 
     // create and verify WebLogic domain image using model in image with model files
-    String imageName = createAndVerifyDomainImage();
+    miiImage = createAndVerifyMiiImage();
 
-    // create and verify one cluster domain
+    // create and verify one cluster mii domain
     logger.info("Create domain and verify that it's running");
-    createAndVerifyDomain(imageName, domain1Uid);
+    createAndVerifyDomain(miiImage, domain1Uid, domain1Namespace, "FromModel", 1);
```

Comment:
Dongbo has refactored a method to use common code, please see if that is useful here.
https://github.com/oracle/weblogic-kubernetes-operator/pull/1759/files#diff-563c7e0923febf5308b79abec0486d46

Reply:
I add request to add domain creation based on custom image name, right now it uses prebuilt image. in my test I have prop files based image creation